src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/NewUser.vue

```diff
@@ -1,38 +1,78 @@
-        </v-form>
-        <v-row>
-          <v-btn rounded="lg" color="primary" class="mr-2" @click="submit">Save and continue</v-btn>
-          <v-btn rounded="lg" variant="outlined" @click="logout">Cancel</v-btn>
-        </v-row>
+    <div class="d-flex flex-column ga-8">
+      <div>
+        <h1>Create your My ECE Registry account</h1>
+        <p class="small mt-2">Welcome {{ oidcUserInfo.firstName!.replace(/^(.*?)(\s.*)?$/, "$1") }}.</p>
+      </div>
+      <div>
+        <ECEHeader title="Information from your BC Services Card account" />
+        <p class="small mt-2">We automatically use the following information from your BC Services Card account for your MY ECE Registry account.</p>
+        <p class="small mt-5">Legal name</p>
+        <p class="small mt-2">{{ oidcUserInfo.firstName }} {{ oidcUserInfo.lastName }}</p>
+
+        <p class="small mt-5">Address</p>
+        <p class="small mt-2">{{ oidcUserInfo.address.street_address }}</p>
+        <p class="small">{{ oidcUserInfo.address.locality }}, {{ oidcUserInfo.address.region }} {{ oidcUserInfo.address.postal_code }}</p>
+        <p class="small">{{ oidcUserInfo.address.country }}</p>
       </div>
-    </FormContainer>
+      <v-form ref="form" validate-on="input">
+        <div class="d-flex flex-column ga-2">
+          <ECEHeader title="Contact Information" />
+          <p class="small mt-2">We'll use this to contact you about your account and updates about your application or certificate.</p>
+          <v-text-field
+            v-model="email"
+            class="mt-5 max-width-500"
+            label="Email"
+            variant="outlined"
+            color="primary"
+            type="email"
+            :rules="emailRules"
+          ></v-text-field>
+          <v-text-field
+            v-model="phoneNumber"
+            label="Phone number"
+            variant="outlined"
+            color="primary"
+            class="max-width-300"
+            :rules="phoneRules"
+            @keypress="isNumber($event)"
+          ></v-text-field>
+          <ECEHeader title="Your ECE registration" />
+          <p class="small mt-2">
+            To add an existing ECE certificate to your account to manage it online, you must enter the number now to link it to your account.
+          </p>
+          <p class="small mt-5">Do you have, or ever had, an ECE certificate in British Columbia?</p>
+          <v-radio-group v-model="eceCertificateStatus" :rules="radioRules">
+            <v-radio label="Yes" :value="true"></v-radio>
+            <v-radio label="No" :value="false"></v-radio>
+          </v-radio-group>
+
+          <v-text-field
+            v-if="eceCertificateStatus === true"
+            v-model="eceRegistrationNumber"
+            label="Your ECE Registration Number"
+            variant="outlined"
+            color="primary"
+            class="max-width-300"
+            :rules="eceRegistrationRules"
+            @keypress="isNumber($event)"
+          />
```

Comment:
AC: 
The input should be a text input, not a numeric one. The number may be too large for a numeric field and has ux issues with accidentally changing the number in a numeric field.

I think we should copy what we have done for other fields that accept numbers. Where we restrict their input. So they won't need that numbers rule. 

Ex. https://github.com/bcgov/ECC-ECER/blob/b4148409b408509130f04fb44fa1d828dfe33323/src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceWorkExperienceReferences.vue#L41

Reply:
I've already added this: @keypress="isNumber($event)" see line 59, I just get rid of this one:

(v: string) => {
          if (this.eceCertificateStatus === true) {
            return /^\d+$/.test(v) || "Your ECE registration number must be a number"; // Must be a number if "Yes" is selected
          }
          return true; // No validation if "No" selected
        },